tslearn/clustering.py

```diff
@@ -384,6 +395,9 @@ def predict(self, X):
         labels : array of shape=(n_ts, )
             Index of the cluster each sample belongs to.
         """
+        X = check_array(X, allow_nd=True)
+        check_is_fitted(self, ['X_fit_'])
```

Comment:
Just FYI: if there's only one attribute, you can use a string instead of an iterable:
```python
check_is_fitted(self, 'X_fit_')
```